### PR TITLE
[moveit_setup_assistant] Added script to pass in environment variables.

### PR DESCRIPTION
**Pull request by tonybaltovski**
### Description
A simple script was added to load in environment variables which are used in the URDF.  When adding 
 a manipulator to a robot, the URDF is extended using an environment variable ([example](https://github.com/ridgeback/ridgeback/blob/indigo-devel/ridgeback_description/urdf/ridgeback.urdf.xacro#L240)) which was not possible prior to these changes.

### Checklist
- [X] **Required**: Code is auto formatted using [clang-format](http://moveit.ros.org/documentation/contributing/code)
- [ ] Extended the tutorials / documentation, if necessary [reference](http://moveit.ros.org/documentation/contributing/)
- [ ] Include a screenshot if changing a GUI
- [ ] Optional: Created tests, which fail without this PR [reference](http://docs.ros.org/kinetic/api/moveit_tutorials/html/doc/tests.html)
- [X] Optional: Decide if this should be cherry-picked to other current ROS branches (Indigo, Jade, Kinetic)
Indigo

### CODE REVIEW

**Comment by v4hn** [comment]
Hey Tony,

could you give a concrete application of this as an example, e.g. in the comment section of the script?

Also, what is the `load_env` param used for? Afaics the script is supposed to export a set of parameters specified in a file to a command it receives as a parameter.
Why would you run it without parameters to set a ros-parameter to the empty-string?

**Comment by tonybaltovski** [comment]
If you setup a workspace with `https://github.com/tonybaltovski/ridgeback_ur5.git` and `https://github.com/ridgeback/ridgeback.git`. Setup the environment to use the extended URDF by `export RIDGEBACK_URDF_EXTRAS=$(catkin_find ridgeback_ur5 urdf/ur5.urdf.xacro --first-only)`.  Launch the moveit_setup_assistant and select `ridgeback_description/urdf/ridgeback.urdf.xacro`.  The UR5 will not show up.  However, switching to this branch would load the environment variables and re-launching the moveit_setup_assistant will show the UR5 attached.

The `load_env` param is used to load the envars into xacro when it is called by moveit_setup_assistant.  I hope this help clear up the use-case.

**Comment by jrgnicho** [comment]
@tonybaltovski  the [xacro](https://github.com/ridgeback/ridgeback/blob/indigo-devel/ridgeback_description/urdf/ridgeback.urdf.xacro) you mentioned does not include the ur5 macro you are trying to load.  In my opinion, the right solution is to create a xacro/urdf which includes the ur5 arm and a fixed joint that attaches the ur5 to the base.

File: moveit_setup_assistant/scripts/env_run [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```
[the remark this answers is not in the document]
Comment (v4hn) [reply moveit_setup_assistant/scripts/env_run]: Personally I don't see why this should be part of MoveIt at the moment.
This is why I didn't look at this further until now...

**Comment by rhaschke** [comment]
@tonybaltovski Why don't you simply load your environment variables before launching ros stuff?
I agree with @v4hn that this shouldn't be part of MoveIt.

**Comment by davetcoleman** [comment]
I'm going to close this 6 month old issue, feel free to re-open if needed

**Pull request closed without merging**